### PR TITLE
host resource

### PR DESCRIPTION
**Pull request by amastbau**
##### Short description:
openshift mtv Host CR

### CODE REVIEW

File: ocp_resources/host.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+from ocp_resources.mtv import MTV
+from ocp_resources.resource import NamespacedResource
+
+
+class Host(NamespacedResource, MTV):
+    """
+    Migration Toolkit For Virtualization (MTV) Host object.
```
Comment (rnetser) [inline ocp_resources/host.py:7]:
```suggestion
    Migration Toolkit For Virtualization (MTV) Host resource.
```

**Review by rnetser — CHANGES_REQUESTED** [review]

File: ocp_resources/host.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            teardown=teardown,
+            yaml_file=yaml_file,
+        )
+        self.host_id = host_id
+        self.ip_address = ip_address
+        self.provider_name = provider_name
+        self.provider_namespace = provider_namespace
+        self.secret_name = secret_name or f"{self.name}-secret"
+        self.secret_namespace = secret_namespace or self.namespace
+        self.condition_message_ready = self.ConditionMessage.HOST_READY
```
Comment (myakove) [inline ocp_resources/host.py:39]: This `self.condition_message_ready` not used and I guess it's net needed, whenever you need it you can call it via the Host resource that was created

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_resources/host.py (hunk 2 above)
Comment (amastbau) [reply ocp_resources/host.py:39]: not sure what you mean
it is in
    def wait_for_condition_ready(self):
        self.wait_for_resource_status(
            condition_message=self.condition_message_ready,
            condition_status=self.Condition.Status.TRUE,
            condition_type=self.Condition.READY,
        )

Comment (amastbau) [reply ocp_resources/host.py:39]: in mtv.py

**Pull request merged**